packages/ocean-api-client/src/errors/BadRequestException.ts

```diff
@@ -0,0 +1,13 @@
+import { ApiException, ApiErrorType } from './ApiException'
+
+export class BadRequestApiException extends ApiException {
+  constructor (message?: string) {
+    super({
+      code: 400,
+      type: ApiErrorType.BadRequest,
+      at: Date.now(),
+      message: message,
+      payload: undefined
```

Comment:
follow the map
https://github.com/DeFiCh/jellyfish/blob/a59989a5e687b8cfefca83a83efcbfb73e7c8535/apps/libs/filters/src/ErrorFilter.ts#L51